### PR TITLE
Added "updatable" package list filter

### PR DESCRIPTION
**Pull request by Josca**

### CODE REVIEW

**Review by semtexzv — COMMENTED** [review]
We can implement the same thing just by adding 
```
Updatable bool  `json:"updatable" query:"json_array_length(spkg.update_data::json) > 0"`
```
to `SystemPackagesAttrs`

**Comment by Josca** [comment]
@semtexzv you were right, done.

File: base/utils/gin.go [hunk 1]
```diff
@@ -27,6 +27,15 @@ func LoadParamInt(c *gin.Context, param string, defaultValue int, query bool) (i
 	return value, nil
 }
 
+func LoadParamBool(c *gin.Context, param string, defaultValue bool) bool {
```
Comment (semtexzv) [inline base/utils/gin.go:30]: Please remove this function, since its not used anywhere.

Comment (Josca) [reply base/utils/gin.go]: done

**Review by semtexzv — APPROVED** [review]

**Pull request merged**